appveyor is unhappy:
```
libzmq-static.dir\Release\precompiled.obj : fatal error LNK1112: module machine type 'x86' conflicts with target machine type 'x64' 
```

--- a/conanfile.py
+++ b/conanfile.py
@@ -26,8 +26,7 @@ class ZMQConan(ConanFile):
             del self.options.fPIC
 
     def build_requirements(self):
-        if not tools.which("ninja"):
-            self.build_requires('ninja_installer/1.8.2@bincrafters/stable')
+        self.build_requires('ninja_installer/1.8.2@bincrafters/stable')
 
     def requirements(self):
         if self.options.encryption == 'libsodium':

--- a/conanfile.py
+++ b/conanfile.py
@@ -27,7 +27,7 @@ class ZMQConan(ConanFile):
 
     def build_requirements(self):
         if not tools.which("ninja"):
-            self.build_requires.add('ninja_installer/1.8.2@bincrafters/stable')
+            self.build_requires('ninja_installer/1.8.2@bincrafters/stable')
 
     def requirements(self):
         if self.options.encryption == 'libsodium':

